hack/generator/pkg/astmodel/type_visitor.go

```diff
@@ -178,12 +178,12 @@ func IdentityVisitOfObjectType(this *TypeVisitor, it *ObjectType, ctx interface{
 func IdentityVisitOfMapType(this *TypeVisitor, it *MapType, ctx interface{}) (Type, error) {
 	visitedKey, err := this.Visit(it.key, ctx)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to visit map key type %T", it.key)
+		return nil, errors.Wrapf(err, "failed to visit map key type %q", it.key.String())
```

Comment:
You don't need the `.String()` calls here: the `%q` formatter will call `.String()` on any `Stringer`.